sql.txt

```diff
@@ -1,14 +1,31 @@
 (1) Using the Customer Table, select the CustomerName, ContactName, and Country.  Use ORDER BY to order by Country.  
-Use LIMIT and OFFSET to get entries 11 through 20. Paste your SQL statement below. 
+Use LIMIT and OFFSET to get entries 11 through 20. Paste your SQL statement below.
+
+SELECT CustomerName, ContactName, Country FROM Customers ORDER BY Country LIMIT 9 OFFSET 10
```

Comment:
You actually want `LIMIT 10` here because the range 11 through 20 is a total of 10 entries (20 - 11 + 1 = 10).

```suggestion
SELECT CustomerName, ContactName, Country FROM Customers ORDER BY Country LIMIT 10 OFFSET 10
```